manuscript/applications.tex

```diff
@@ -0,0 +1,36 @@
+\section{Other Uses of Dynamic Deployment}
+The concept of dynamic demand-driven deployment is not new,
+and has been used in various applications. Various algorithms
+are used to optimize deployment of facility or utilization
+of given sources.
+
+\subsection{Aircraft Dispatch}
+To maximize fleet utilization and minimize
+operating costs, airlines predict future demands
+and optimize their flight schedule and aircraft
+type using linear programming methods \cite{berge_demand_1993}. The Demand
+Driven Dispatch process evaluates benefits and costs
+in various stages of an airline planning stage, like
+pricing, aircraft assignment, and crew scheduling \cite{shebalov_practical_2009}.
+
+\subsection{Timber Industry}
+A sawmill facility that processes timber contains 
+many steps, from sawing, drying to planning. An
+agent-based simulation is used to analyse demand-
+driven production planning \cite{yáñez_agent-based_2009}.
+The facility unit can be represented by a group of many facilities,
+for example, source, sawing operations, drying operations, warehouse,
+make, and delivery. In the simulation, each agents are given their parameters
```

Comment:
Unfortunately it doesn't make more sense. I've just read through this part of the paper at hand and I think the interpretation represented here is wrong. Sawing, drying, warehouse, and finishing are different types of "Make Agents". "Make Agent" is a category of agents (like a virtual class, similar to our notion of Facility models). It is never a concrete instance of itself. It's merely a genre of agent, defined by the fact that it makes demand (for raw material) and offers a supply (of post-process material).